cmd/config_envs.go

```diff
@@ -25,20 +28,20 @@ Prints configured Environment variable for a function project present in
 the current directory or from the directory specified with --path.
 `,
 		SuggestFor: []string{"ensv", "env"},
-		PreRunE:    bindEnv("path"),
+		PreRunE:    bindEnv("path", "output"),
 		RunE: func(cmd *cobra.Command, args []string) (err error) {
-			function, err := initConfigCommand(args, defaultLoaderSaver)
+			function, err := initConfigCommand(loadSaver)
 			if err != nil {
 				return
 			}
 
-			listEnvs(function)
-
-			return
+			return listEnvs(function, cmd.OutOrStdout(), Format(viper.GetString("output")))
 		},
 	}
 
-	configEnvsAddCmd := NewConfigEnvsAddCmd()
+	cmd.Flags().StringP("output", "o", "human", "Output format (human|json) (Env: $FUNC_OUTPUT)")
```

Comment:
In the other commands I decided to simplify this to just a single flag, `--json`.  I know that means it may be tricky in the future if we add several other output formats, but it sure seems to make the CLI experience nicer now:  For example
```
func templates --json
```
I found that to offer a better experience.  I therefore prefer it that way, personally, but either way we choose, we should probably be consistent.  Thoughts?

Reply:
+1 to `-o`

Reply:
I will use `-o`.

`-ojson` is the same length as `--json`.